scalding-core/src/main/scala/com/twitter/scalding/filecache/DistributedCacheFile.scala

```diff
@@ -7,7 +7,7 @@ import java.net.URI
 import java.nio.ByteBuffer
 
 import org.apache.hadoop.conf.Configuration
-import org.apache.hadoop.mapreduce.filecache.{DistributedCache => HDistributedCache}
+import org.apache.hadoop.mapreduce.filecache.{ DistributedCache => HDistributedCache }
```

Comment:
Ah, so this is what happens when you run the unit tests (scalafmt) -- I wasn't sure whether to keep this. If I don't commit this, next time someone runs the unit tests, it'll do this again. So I thought I'd just keep the changes. I can back them out though and send a separate PR that only does the style fixes.